### ISSUE TITLE
Ferdi 5.4.4-beta.2 with Dark mode crashes on Reddit

### ISSUE THREAD

**Issue by alexdevero** [getferdi/ferdi#422]
**Describe the bug**
The latest version of Ferdi (5.4.4-beta.2) crashes on Reddit, few seconds after clicking on any post.

**To Reproduce**
Use my custom recipe for [Reddit](https://github.com/alexdevero/franz-reddit-recipe).

1. Create new service using Reddit custom recipe.
2. Enable Dark mode in app settings.
2. Open Reddit service.
3. Click on some post.
4. After a few seconds Ferdi crashes.

**Expected behavior**
Ferdi will not crash.

**Desktop:**
- OS: Windows 10
- Ferdi Version: 5.4.4-beta.2

Debug information: https://debug.getferdi.com/e0b84bb3-5226-49f2-bd3a-7cdd0ad9dcd6

**Additional context**
1) I am using Ferdi 5.4.4-beta.2 portable
2) My Reddit user account is set to dark mode, like Ferdi app itself.
3) If I disable Dark mode for Reddit service, Ferdi no longer crashes.

**Comment by vantezzen** [comment]
Can't reproduce on macOS. I've done the following:
1. Install your recipe and create a new service with it
2. Open a post in reddit - everything works fine
3. Enable Dark mode in Ferdi (incl. universal dark mode)
4. Open a post in reddit - everything works fine
5. Login to Reddit and enable Night Mode
6. Open a post in reddit - everything works fine
7. Restart Ferdi
8. Open a post in reddit - everything works fine

**Comment by alexdevero** [comment]
@vantezzen Weird. Did you find any information about recent crashes in the debug log?

**Comment by vantezzen** [comment]
No, the debug log doesn't contain information about service content (like console errors) due to privacy reasons.

**Comment by alexdevero** [comment]
The crash happened three times. I can't reproduce it after the last restart of Ferdi.

**Comment by alexdevero** [comment]
I just experienced another 3 crashes. Two when opening a post and 1 when trying to use filter input at the top.

**Comment by alexdevero** [comment]
There are many warnings about failing to parse SourceMap. Could this cause the crash, something like overflowing the app with too many messages/requests?

**Comment by vantezzen** [comment]
The warnings about the SourceMaps are normal - I see them on my non-crashing service too - and shouldn't be able to crash the app.

**Comment by alexdevero** [comment]
Another three crashes. Is there some crash logger built-in in Ferdi, or electron?

**Comment by kytwb** [comment]
@alexdevero could you please opt-in for the telemetry data so we may get a Sentry report for the crash you're experiencing?

<img width="596" alt="Screen Shot 2020-03-19 at 2 51 58 AM" src="https://user-images.githubusercontent.com/412895/77023600-9e46cf00-698c-11ea-8071-248abd30ab46.png">

**Comment by alexdevero** [comment]
@kytwb Yes, sure.
